payroll-processor-client/import-sorter.json

```diff
@@ -0,0 +1,30 @@
+{
+  "groupRules": [
```

Comment:
I had forgotten to add this to the other configuration PRs.

This file works with the `dozerg.tsimportsorter` extension to define how imports get organized and sorted.

It's currently set up to create 5 groups of imports.

1. Angular/RxJs imports
2. 3rd part library imports
3. TSConfig path alias imports (when importing modules from other parts of the app)
4. Relative parent path modules
5. Relative sibling path modules

Whenever we add a new 3rd party lib, we can add it to the `^@datorama|@fortawesome|@ng-bootstrap` list to ensure it gets sorted correctly.

Reply:
I never knew we could customize imports at the project level. Nice!